src/display/views/threads/components/__tests__/CheckboxTable.spec.js

```diff
@@ -81,7 +81,7 @@ describe('behavior', () => {
 				}
 			]);
 		});
-		it('should trigger onSelectionChanged when item is added', () => {
+		it('should trigger onSelectionChanged when item is removed', () => {
```

Comment:
This was a duplicate test name. Updated to what I believe the intent was. Ditto for L#242 below

Reply:
Nice catch! Thanks for fixing. :)